launch_ros/launch_ros/actions/node.py

```diff
@@ -237,13 +261,19 @@ def parse(cls, entity: Entity, parser: Parser):
         node_name = entity.get_attr('node-name', optional=True)
         if node_name is not None:
             kwargs['node_name'] = parser.parse_substitution(node_name)
+        node_name = entity.get_attr('name', optional=True)
+        if node_name is not None:
+            kwargs['name'] = parser.parse_substitution(node_name)
```

Comment:
If both `name` and `node_name` are specified, it should be detected as an error.

Reply:
This is detected by fa4ab9f